### DISCUSSION TITLE
Unit conversion of mesh between the Reactor module and the THM module

### DISCUSSION THREAD

**Discussion by NorthMagic** [idaholab/moose#27430] [Q&A Modules: Thermal Hydraulics] [answered]
Hello, everyone! I'm using THM's `HeatStructureFromFile3D`, in which the mesh file is generated by Reactor module. Unfortunately, the unit of mesh from Reactor module is `cm` but that of THM module is `m`. I don't know how to do the unit conversion between `HeatStructureFromFile3D` (mesh file from Reactor module: cm) and `FlowChannel1Phase` (mesh from THM module: m). Could you please give me some suggestions?

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can use the coordinate transformation system to perform the conversion between units 

The parameters to set are in the mesh block
More details can be found here on the Mesh syntax page
[Coordinate Systems](https://mooseframework.inl.gov/syntax/Mesh/index.html#5fd67a6c-376e-4b53-bd33-840af95c24c8)

Guillaume

**Reply by NorthMagic** [reply]
Thanks for your help, it was fixed.
